tests/http/federation/test_matrix_federation_agent.py

```diff
@@ -49,11 +49,16 @@ def setUp(self):
 
         self.mock_resolver = Mock()
 
+        config = Mock()
```

Comment:
I'd suggest using `tests.utils.test_config` rather than making your own Mock which may or may not correctly match the shape of a real Config.